uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/setting/SettingViewModel.kt

```diff
@@ -0,0 +1,41 @@
+package io.github.droidkaigi.feeder.setting
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.compositionLocalOf
+import io.github.droidkaigi.feeder.AppError
+import io.github.droidkaigi.feeder.Theme
+import io.github.droidkaigi.feeder.core.UnidirectionalViewModel
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.StateFlow
+
+interface SettingViewModel :
+    UnidirectionalViewModel<SettingViewModel.Event, SettingViewModel.Effect, SettingViewModel.State> {
```

Comment:
Looks like we have a ktlint failure here due to chars are more than 100 in a line.
`Exceeded max line length (100)`
You can simply break them into multiple lines to pass the GitHub checks.

Reply:
@anandwana001 
Thanks, I fixed it.